docs/src/community/phrasebook.rst

```diff
@@ -0,0 +1,67 @@
+.. include:: ../common_links.inc
+
+.. _phrasebook:
+
+Package Phrasebook
+===================
+
+There are a number of similar packages to Iris, and a lot of these have their own
+terminology for similar things. Whether you're coming or going, we hope this might
+be a helpful guide to these differences!
+Definitions for each can be found in :ref:`glossary`. See also
+`Xarray terminology <https://docs.xarray.dev/en/stable/user-guide/terminology.html>`_.
+
+.. list-table:: Phrasebook
+   :widths: 25 25 25 50
+   :header-rows: 1
+
+   * - Iris
+     - Xarray
+     - Example
+     - Notes
+   * - Non-Lazy
+     - Eager
+     -
+     - Used to relate to functions, rather than the data.
+   * - Cube
+     - DataArray
+     -
+     -
+   * - CubeList
+     - Dataset
+     -
+     - Though similar, a CubeList is a simpler object, and is
+       not a perfect comparison to a Dataset
+   * - Merge/ Concatenate
+     - Concatenate
+     - `Xarray concatenate <https://docs.xarray.dev/en/stable/user-guide/combining.html#concatenate>`_
+     - Xarray's concatenate has the capability to largely do what both
+       iris merge and iris concatenate do, this is not a perfect comparison,
+       please see the link for more information.
+   * -
+     - Merge
+     - `Xarray merge <https://docs.xarray.dev/en/stable/user-guide/combining.html#merge>`_
+     - Xarray's Merge function doesn't map neatly map to any iris feature.
+       Please see the link for more information.
+   * -
+     - Scalar
+     -
+     - Iris makes a distinction between scalar coordinates and non-scalar
+       coordinates, xarray does not. In xarray, a scalar is any non-array or
+       array-like object e.g. int, float or str.
```

Comment:
I'm still not entirely sure about the escond sentance here. I feel like the concept of a "scalar" described here is something that is implicit in Iris as well so this isn't really a unique xarray term. I feel like the real difference worth highlighting is the fact that while Iris documentation attaches the scalar label to the coordinate, xarray documentation applies the scalar label to the data. I also think it's worth making it clear that it is possible to make coordinates with scalar data in both iris and xarray which are effectively equivalent. This may be getting a bit wordy, but maybe something like:

Iris makes a distinction between scalar coordinates and non-scalar coordinates, xarray documentation makes a distinction between scalar and non-scalar __data__. It is possible to make coordinates with scalar data in both iris and xarray but only iris will label such coordinates.